docs/apis-clients/working-with-apis-clients.md

```diff
@@ -1,5 +1,5 @@
 ---
-id: working-with-apis-tools
+id: working-with-apis-clients
```

Comment:
I could have left the id as "working-with-apis-tools", but I feel like consistency between the ID and filename is less likely to lead to confusion.